ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,52 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+
+### Segment Routing Summary
+
+- SRTE is enabled.
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+
+- system-colored-tunnel-rib is enabled
+{%         endif %}
+
+### SRTE Policies
```

Comment:
Increase this heading level to ```####```. This gives us an advantage to group SRTE Policies and other Segment Routing headings or tables (in future development) under ```Segment Routing Summary``` . Also other ```Router Traffic-Engineering``` settings will be using the similar technique to group under their respective heading.